x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/timeline/data_providers.ts

```diff
@@ -34,7 +29,7 @@ describe('timeline data providers', () => {
     dragFromAllHostsToTimeline();
 
     cy.get(TIMELINE_DROPPED_DATA_PROVIDERS, {
-      timeout: TIMELINE_RENDER_DATA_PROVIDERS_TIMEOUT,
+      timeout: DEFAULT_TIMEOUT + 10 * 1000,
```

Comment:
Ran it a few times but still got: `Timed out retrying: Expected to find element: '[data-test-subj="all-hosts"] [data-react-beautiful-dnd-drag-handle]', but never found it.` :(

Reply:
I'm doing some ad-hoc testing of another PR and i'm hitting timeouts in the app itself. It looks like `siem-dev` is not healthy ATM.